### PR TITLE
Fixing L0_http and L0_grpc

### PR DESCRIPTION
**Pull request by tanmayv25**
@CoderHam  This should fix the model control. It broke because we missed modifying the regex in http in PR #1509.

### CODE REVIEW

File: src/clients/python/examples/grpc_image_client.py [hunk 1]
```diff
@@ -64,6 +65,17 @@ def model_dtype_to_np(model_dtype):
         return np.dtype(object)
     return None
 
+def deserialize_bytes_tensor(encoded_tensor):
```
Comment (CoderHam) [inline src/clients/python/examples/grpc_image_client.py:68]: This is already available in the utils package

Comment (tanmayv25) [reply src/clients/python/examples/grpc_image_client.py:68]: This is to be served as an example if users want to bypass our client package. Hence, I included the deserialization logic here itself. I would avoid using the client packages here.

File: src/clients/python/examples/grpc_image_client.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 from PIL import Image
 import os
 import sys
+import struct
 
 import grpc
 from tritongrpcclient import grpc_service_pb2
```
Comment (CoderHam) [inline src/clients/python/examples/grpc_image_client.py:36]: the imports can be comma separated such as `from a import x, y`

Comment (tanmayv25) [reply src/clients/python/examples/grpc_image_client.py]: Done

**Pull request merged**